data_dir/latte/schema_scale.rn

```diff
@@ -0,0 +1,277 @@
+        "col_date": (hash_range(data_gen_seed, 4_294_967_295) + 1_000_000_000) % 4_294_967_295,
+        "col_blob": blob(data_gen_seed, BLOB_COL_SIZE),
+        "col_names": [ // utility attr to be used by further data verification
+            "col_text",
+            "col_date",
+            "col_blob",
+        ]
+    };
+    ret
+}
+
+async fn get_ks_table_mapping_i(db, i) {
+    let shift_window_i = i % TABLE_GROUP_SIZE;
+    if TABLE_SHIFT_INTERVAL_S > 0 {
+        let elapsed = db.elapsed_secs();
+        let shift_window_offset = (elapsed as i64 / TABLE_SHIFT_INTERVAL_S) * TABLE_SHIFT_COUNT;
+        let ks_table_mapping_i = (shift_window_offset + shift_window_i) % db.data.total_tables;
+        ks_table_mapping_i
+    } else {
+        shift_window_i
+    }
+}
+
+pub async fn write(db, i) {
+    let ks_table_mapping_i = get_ks_table_mapping_i(db, i).await;
+    let ks_index = db.data.ks_table_mapping[ks_table_mapping_i][0];
+    let t_index = db.data.ks_table_mapping[ks_table_mapping_i][1];
+    // NOTE: each table getting queries after a table shift will get non-zero first row index.
+    //       It is ok because we care about the step increment value and not starting index.
+    let row_i = (i / TABLE_GROUP_SIZE) % ROW_COUNT_PER_TABLE;
+    let idx = get_idx(db, row_i).await;
+
+    let pk = uuid(idx);
+    let data = gen_data(idx, ks_index, t_index).await;
+    db.execute_prepared(db.data.p_stmt[ks_index][t_index].INSERT, [
+        pk, data.col_text, data.col_date, data.col_blob,
+    ]).await?
+}
+
+pub async fn read(db, i) {
+    let ks_table_mapping_i = get_ks_table_mapping_i(db, i).await;
+    let ks_index = db.data.ks_table_mapping[ks_table_mapping_i][0];
+    let t_index = db.data.ks_table_mapping[ks_table_mapping_i][1];
+    // NOTE: each table getting queries after a table shift will get non-zero first row index.
+    //       It is ok because we care about the step increment value and not starting index.
+    let row_i = (i / TABLE_GROUP_SIZE) % ROW_COUNT_PER_TABLE;
+    let idx = get_idx(db, row_i).await;
+
+    let pk = uuid(idx);
+    if !DATA_VALIDATION {
+        db.execute_prepared(db.data.p_stmt[ks_index][t_index].SELECT, [pk]).await?
+    } else {
+        let rows = db.execute_prepared_with_result(db.data.p_stmt[ks_index][t_index].SELECT, [pk]).await?;
+        let rows_len = rows.len();
+        let row_err_info = (
+            `ks='${db.data.p_stmt[ks_index][t_index].keyspace}', ` +
+            `table='${db.data.p_stmt[ks_index][t_index].table}', pk='${pk}'`);
+        if rows_len != 1 {
+            // NOTE: may be false negative only if we didn't populate DB correctly.
+            db.signal_failure(
+                `Expected exactly 1 row. But got '${rows_len}'. ` + row_err_info
+            ).await?;
+        }
+        let data = gen_data(idx, ks_index, t_index).await;
+        for col_name in data.col_names {
+            // TODO: make it be compatible with prep stmts which select non-all columns
```

Comment:
raise issue for it, as followup for this one

Reply:
Moved the rune script into its own PR:
- https://github.com/scylladb/scylla-cluster-tests/pull/12954

Also, the rune script is not merged, so, there is no required condition to file a QA task